test/wrapper_test.rb

```diff
@@ -3060,7 +3060,7 @@ def test_detecting_unfeasible_services_can_not_take_too_long
       }
     end
 
-    assert_operator total_time, :<=, 1.5, 'check_distances function took longer than expected'
+    assert_operator total_time, :<=, 15, 'check_distances function took longer than expected'
```

Comment:
@fonsecadeline did you check the performance of this test? I suspect this modification might be incorrect. If we are losing 10x perf, there is a regression somewhere.

Maybe you didn't realize the test has changed? We now set the repetition to 1 and set the log level to fatal. So the performance is stable and 1 repeat decreases the time. This test passed regularly on local and on travis for mapotempo/dev when I was pushing for maj-fixes. 
```
    OptimizerLogger.level = :fatal # turn off output completely no matter the setting
    OptimizerWrapper.config[:solve][:repetition] = 1 # fix repetition to measure the perf correctly
```

Reply:
I realized the test changed but I did not realize limit has also changed (which makes sense ...)! This is an old PR and we used to have an issue on travis because previous version (with previous limit) was too strict so I thought I still needed to fix it. 
But you are right, the tests passes everytime on local so great ! 

I will revert this modif then :)